CHANGELOG.md

```diff
@@ -26,7 +26,7 @@ Monitor the release status by regions at [AKS-Release-Tracker](https://releases.
   * Fixed an issue where PUT operations on managedClusters or agentPools see long latency in the overall operation due to an internal network issue.
   * PATCH operations were allowed on managedClusters in a non-terminal provisioningState.  This could cause an eTag mismatch and inconsistent results or failures.  PATCH operations will now be block for managedClusters in a non-terminal provisioningState.
 * Behavioral Change
-  * Updates to optimize the kube-reserved eviction thresholds [available in 1.28](https://learn.microsoft.com/en-us/azure/aks/concepts-clusters-workloads#memory)
+  * Changes to reduce the kube-reserved memory reservation and eviction threshold will not be available in 1.28 due to a release issue. These optimizations will be releasing with AKS 1.29, which previews in January 2024.
```

Comment:
Changes to reduce the kube-reserved memory reservation and eviction threshold will not be available in 1.28 as previously shared due to a release issue. These optimizations will be releasing with AKS 1.29, which previews in January 2024